cli/cli_connect.go

```diff
@@ -33,14 +33,14 @@ Provide a <group> argument to connect to a specific servers group. For example:
 Press the Tab key to see auto-suggestions for countries and cities.`
 )
 
-func (c *cmd) browseToSubscriptionPage() {
+func (c *cmd) browseToSubscriptionPage(url string, loginURL string) {
 	resp, err := c.client.TokenInfo(context.Background(), &pb.Empty{})
 	if err != nil {
-		browse(client.SubscriptionURL)
+		browse(url, url)
 		return
 	}
 
-	browse(fmt.Sprintf(client.SubscriptionURLLogin, resp.Token, resp.Id))
+	browse(fmt.Sprintf(loginURL, resp.Token, resp.Id), url)
```

Comment:
This is not something new, but is it ok if the login token is passed in plain text part of the URL? Aren't there any security issues?

Reply:
Yeah, I misunderstood the spec a bit, actually token used in the URL is supposed to be a Trusted Pass token, which is supposed to be safe.